src/ontology/imports/axioms.owl

```diff
@@ -144,7 +144,7 @@
         <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/SO_0000704"/>
         <rdfs:subClassOf>
             <owl:Restriction>
-                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/RO_0002573"/>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/RO_0000053"/>
```

Comment:
@twhetzel this change is the only thing that matters. History:

A while back, we switched all "inherited" diseases to using "has characteristic" some "inherited". This is a super critical GCI that ensures that all diseases that have their material basis in some germline mutation are automatically classified as "inherited". This PR will have a huge diff on the class hierarchy, it will have definitely contributed to additional work on the "hereditary diseases" branch during mass obsoletion and should simply never have happened. Steps forward:

1. Crucify me
2. Ensure this never happens again (requires a bit of thinking, as this is not easy)